### DISCUSSION TITLE
PLEX

### DISCUSSION THREAD

**Discussion by VerVRHUB** [liuw086/SuperScreen#63] [General]
Hi, I'm looking for a Quest 3 app that allows me to see my PLEX library without enabling the DLNA function (since with DLNA it is not possible to set a password to protect the media).
I've heard that SUPERSCREEN includes PLEX but I have a couple of questions about SUPERSCREEN that I'm hoping someone can answer:
1- which streaming apps does it support (PRIME, NETFLIX, PLEX etc.)?;
2- If PLEX is included in this list. Does it allow me to watch videos from my library (PC) as happens with smartTVs or firesticks, via LAN even without enabling DLNA? ;
3- If PLEX is included, does it allow viewing of contents at their maximum resolution?

For now thanks,
Ver

**Comment by liuw086** [comment]
SuperScreen application only supports access to https://www.plex.tv/ through the built-in browser. You mentioned accessing files on your computer. I am not sure whether this is a function provided by the plex client application. . If you cannot access your PC library when accessing plex through a browser(Chrome) on your computer, you should not be able to access it on SuperScreen either.

**Comment by VerVRHUB** [comment]
Thenk's liuw. to https is great.
Another question is: Using PLEX, SuperScreen may can show also SDB 3d movies?

**Reply by liuw086** [reply]
The browser HTTP mode does not currently support SBS 3d, I still have to research. Currently only local or LAN video support
